### PR TITLE
Resolves #1466

### LINKED ISSUE #1466: Add state/US-level incident case forecasts to viz
Right now the visualization does not show incident case forecasts. Some teams have already submitted incident case forecasts to their submitted data. We also already have incident case ground truth data in the repository. This is just a matter of adding the required data processing to show incident case data.

### PR DESCRIPTION
**Pull request by ydhuang28**
## Description

Incident cases forecasts can now be seen on the visualization.

### CODE REVIEW

File: visualization/vis-master/covid-csv-tools/dist/meta.js [hunk 1]
```diff
@@ -68,6 +68,12 @@ exports.targetFullNameCum = {
   '3-ahead': '3 wk ahead cum death',
   '4-ahead': '4 wk ahead cum death'
 };
+exports.targetFullNameIncCase = {
```
Comment (hannanabdul55) [inline visualization/vis-master/covid-csv-tools/dist/meta.js:71]: Is this variable used anywhere?

**Review by hannanabdul55 — APPROVED** [review]
Looks good! One minor suggestion, I think with this change, the default target chosen when you load the viz is `inc cases`, this is because the default target is set to the last one (`size_of_seasons-1`) when loaded, could you change that to the Cumulatve deaths target? It's in `Chloropeth.vue#ready()` function. Otherwise this looks good to me and let's get this merged! 👍

**Comment by nickreich** [comment]
Is @hannanabdul55 's comment above about which of the forecasted targets are loaded when the page first loads? if so, I might prefer to have this load the incident deaths by default instead of cumulative deaths. minor point, and may warrant an issue of it's own?

**Comment by hannanabdul55** [comment]
@nickreich I think this is straightforward to do here itself. Then it is even easier, just load the season in the above mentioned function to `0`.

File: visualization/vis-master/covid-csv-tools/dist/meta.js (hunk 1 above)
Comment (ydhuang28) [reply visualization/vis-master/covid-csv-tools/dist/meta.js:71]: Yep, it's used down below to pick out the incident case data from the filtered CSV files.

Comment (ydhuang28) [reply visualization/vis-master/covid-csv-tools/dist/meta.js:71]: Maybe because it has `csv` in the filename?

**Comment by ydhuang28** [comment]
Hmm. I had the same idea and tried that, but apparently the data won't load on start-up if I update the selected target to `0`. Both `1` (Cumulative Deaths) and `2` (Incident Cases) load the data correctly but `0` doesn't. However, if I start with Cumulative Deaths or Incident Cases and switch to Incident Deaths it's totally fine. Interestingly, if I don't call `updateSelectedSeason()`, `this.selectedSeasonId` defaults to 0; I wonder this is why the `updateSelectedSeason()` was added? @hannanabdul55 any thoughts?

**Comment by ydhuang28** [comment]
I'll keep working on this in the mean time.

**Comment by hannanabdul55** [comment]
@ydhuang28 yeah, this seems like a weird behavior. I think it is because of the season download data that happens at load just download only the last one? I am not sure exactly what's going on there. Will need to dig into that more. But I think that is not a big issue right now and could be tracked in another Github issue itself and in the meantime, we could maybe merge this? I am fine with either way.

**Comment by ydhuang28** [comment]
Yep, we can merge this. I'll open a separate issue for the data loading problem.

**Comment by ydhuang28** [comment]
I've created a new issue for showing the correct default target: [Issue #1536](https://github.com/reichlab/covid19-forecast-hub/issues/1536)

**Pull request merged**